### PR TITLE
Add unicopedia-plus

### PR DESCRIPTION
**Pull request by dahlia**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@dahlia Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by dahlia** [comment]
@bevanjkay Added a `zap` stanza.  Please take a look again!

**Comment by dahlia** [comment]
@bevanjkay @carlocab @miccal  Friendly ping.  Could anybody review this for me?

**Comment by bevanjkay** [comment]
@dahlia It looks like the tests are failing at the moment.

**Comment by dahlia** [comment]
@bevanjkay Sorry to bother you!  Apparently I made a mistake during rebasing the commit.  I believe it's no problem now.  Could you take a look again?

File: Casks/unicopedia-plus.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "unicopedia-plus" do
+  version "11.2.1"
```
Comment (bevanjkay) [inline Casks/unicopedia-plus.rb:2]: Update to latest version

File: Casks/unicopedia-plus.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 "396feb5cad7cb8f38d150e86c244ee5310b559dfc7f87d13ce9fa97648727672"
+
+  url "https://github.com/tonton-pixel/unicopedia-plus/releases/download/v11.2.2/unicopedia-plus-mac-universal-11.2.2.dmg"
```
Comment (bevanjkay) [inline Casks/unicopedia-plus.rb:5]: @dahlia This should still use interpolation to provide the version dynamically

```
  url "https://github.com/tonton-pixel/unicopedia-plus/releases/download/v#{version}/unicopedia-plus-mac-universal-#{version}.dmg"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/unicopedia-plus.rb (hunk 1 above)
Comment (dahlia) [reply Casks/unicopedia-plus.rb]: Oh I missed that new release.  Fixed, thanks!

File: Casks/unicopedia-plus.rb (hunk 2 above)
Comment (dahlia) [reply Casks/unicopedia-plus.rb]: Thank you for letting me the better way.  Thanks, I fixed!

**Review by bevanjkay — APPROVED** [review]

**Comment by dahlia** [comment]
Thank you for reviewing!

**Pull request merged**